kolibri/core/assets/src/api_resource.js

```diff
@@ -41,19 +41,13 @@ class Model {
   constructor(data, resource) {
     // Assign any data to the attributes property of the Model.
     this.attributes = {};
-    Object.assign(this.attributes, data);
+    this.set(data);
```

Comment:
This line needs to be swapped with the one below it, because `this..set` references `this.resource`